Raven.Client.Lightweight/Convention.cs

```diff
@@ -29,9 +29,9 @@ public FailoverBehavior FailoverBehaviorWithoutFlags
         }
 
         /// <summary>
-        /// Enable multipule async operations
+        /// Enable multiple async operations
         /// </summary>
-        public bool AllowMultipuleAsyncOperations { get; set; }
+        public bool AllowMultipleAsyncOperations { get; set; }
```

Comment:
Hmm... I don't see separate branch for that, so I assume you want me to send same PR simply later in time, is that correct?
Is there any roadmap and dates when 4.x release is scheduled?